code/game/machinery/doors/airlock.dm

```diff
@@ -1725,6 +1736,29 @@
 	var/area/source_area = get_area(src)
 	return source_area?.airlock_wires ? new source_area.airlock_wires(src) : new /datum/wires/airlock(src)
 
+
+/obj/structure/fluff/airlock_filler/CanAllowThrough(atom/movable/mover, turf/target)
```

Comment:
Even if this doesn't have any modifications to vars please define it on the line aboves separate from its proc definitions, and preferably add a documenting comment `/// or /* */` about what it is for